client/src/components/Info.jsx

```diff
@@ -1,35 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
You could change this line to just import React from 'react' since it's no longer a class component